fe/fe-core/src/main/java/com/starrocks/http/rest/RestBaseAction.java

```diff
@@ -197,4 +219,42 @@ public boolean redirectToLeader(BaseRequest request, BaseResponse response) thro
                 new TNetworkAddress(leaderIpAndPort.first, leaderIpAndPort.second));
         return true;
     }
+
+    protected static String getParameter(BaseRequest request, String paramName) {
```

Comment:
while these `getParameters` are good help functions, thinking about a few improvements,

* `getParameter()` return null if the paremter doesn't exist, it is up to the caller to determine what to do.
* `getParameterRequired()` throws BAD_REQUEST http exception if the parameter is not there.
* `getParameterOrDefault()` returns default value if the parameter is not there.
* generic programming, getParameter for int/boolean/string/double/..., 
org.apache.hadoop.conf.Configuration might be a good reference for these interfaces.

Reply:
@kevincai Good suggestions.